src/structures/MessageEmbed.js

```diff
@@ -11,7 +11,24 @@ class MessageEmbed {
    * @name MessageEmbed
    * @kind constructor
    * @memberof MessageEmbed
-   * @param {MessageEmbed|Object} [data={}] MessageEmbed to clone or raw embed data
+   * @param {MessageEmbed|MessageEmbedOptions} [data={}] MessageEmbed to clone or raw embed data
```

Comment:
```suggestion
   * @param {MessageEmbed|MessageEmbedOptions} [data] MessageEmbed to clone or raw embed data
```
I believe there is no need to include a default empty object since this parameter is already optional. I wasn't sure whether you would want this to be changed so I didn't initially but please let me know if I should add this to the PR

Reply:
It is needed, since it says the value is optional, and by default its an empty object

Reply:
@vladfrangu the value is optional because of the [], it doesn’t need a default empty object and this was done multiple times across the library.

Reply:
It is in fact very inconsistent. You can also check this where data is optional and there is no default value because it doesn’t need one since it isn’t changing anything https://github.com/discordjs/discord.js/blob/stable/src/structures/MessageAttachment.js#L12